src/main/java/frc/robot/RobotContainer.java

```diff
@@ -115,105 +115,55 @@ private void configureBindings() {
+        .whileTrue(new IntakeGamePiece(subIntake));
 
-    // Prep Place; Will be rebound to Left Trigger
-    // conOperator.btn_LeftTrigger.whileTrue(Commands.run(() ->
-    // subArm.setGoalAnglesFromNumpad()).repeatedly());
+    // prep place (x)
+    conOperator.btn_X.onTrue(Commands.runOnce(() -> subArm.setStateFromDesiredNode()));
 
-    // Place Game piece; Will be rebound to Right Trigger
+    // Place Game piece (rt)
     conOperator.btn_RightTrigger.whileTrue(new PlaceGamePiece(subArm, subIntake));
 
-    // // Set Collector to starting config and stop the rollers
-    // conOperator.btn_North
-    // .onTrue(Commands.runOnce(() ->
-    // subCollector.setGoalPosition(prefCollector.pivotAngleStartingConfig)));
-
-    // // Set Collector rollers to intake height and spin the rollers
-    // conOperator.btn_South
-    // .onTrue(Commands.runOnce(() ->
-    // subCollector.setGoalPosition(prefCollector.pivotAngleCubeCollecting)));
-
     // Spin the Intake forward
     conOperator.btn_Start
-        .whileTrue(Commands.run(() -> subIntake.setMotorSpeed(prefIntake.intakeIntakeSpeed), subIntake));
+        .whileTrue(new IntakeGamePiece(subIntake));
 
-    // Spin the Intake in reverse
+    // Spin the Intake in reverse (back)
     conOperator.btn_Back
-        .whileTrue(
-            Commands.run(() -> subIntake.setMotorSpeed(prefIntake.intakeReleaseSpeed), subIntake));
-
-    // Numpad
-    // conNumpad.btn_1.onTrue(Commands.runOnce(() -> subArm.scoringGrid =
-    // ScoringGrid.GRID_1));
-    // conNumpad.btn_2.onTrue(Commands.runOnce(() -> subArm.scoringGrid =
-    // ScoringGrid.GRID_2));
-    // conNumpad.btn_3.onTrue(Commands.runOnce(() -> subArm.scoringGrid =
-    // ScoringGrid.GRID_3));
-
-    // conNumpad.btn_12.onTrue(Commands.runOnce(() -> {
-    // subArm.scoringLevel = ScoringLevel.HYBRID;
-    // subArm.scoringButton = ScoringButton.FIRST;
-    // }));
-
-    // conNumpad.btn_11.onTrue(Commands.runOnce(() -> {
-    // subArm.scoringLevel = ScoringLevel.HYBRID;
-    // subArm.scoringButton = ScoringButton.SECOND;
-    // }));
-
-    // conNumpad.btn_10.onTrue(Commands.runOnce(() -> {
-    // subArm.scoringLevel = ScoringLevel.HYBRID;
-    // subArm.scoringButton = ScoringButton.THIRD;
-    // }));
+        .whileTrue(subIntake.releaseCommand());
+
+    // numpad
 
     // mid cone
     conNumpad.btn_9.onTrue(Commands.runOnce(() -> {
       subArm.setDesiredNode(4);
     }));
 
-    // conNumpad.btn_8.onTrue(Commands.runOnce(() -> {
-    // subArm.scoringButton = ScoringButton.FIFTH;
-    // subArm.scoringLevel = ScoringLevel.MID;
-    // }));
-
     // mid cube
     conNumpad.btn_7.onTrue(Commands.runOnce(() -> {
       subArm.setDesiredNode(5);
     }));
 
+    // high cube
+    conNumpad.btn_4.onTrue(Commands.runOnce(() -> {
+      subArm.setDesiredNode(2);
+    }));
     // high cube
     conNumpad.btn_6.onTrue(Commands.runOnce(() -> {
       subArm.setDesiredNode(1);
     }));
```

Comment:
Are these both high cube?

Reply:
no, node 1 is cone. comment is incorrect

Reply:
resolved in latest commit